### PR TITLE
Fix the catchoutput errors

### PR DESCRIPTION
**Pull request by ThomasLecocq**
* solves the import of obspy from Jupyter Notebooks (https://github.com/obspy/obspy/issues/1866)
* solves part of the catch output complexity (https://github.com/obspy/obspy/pull/1684)

### CODE REVIEW

**Comment by ThomasLecocq** [comment]
No appveyor tests ?

**Comment by ThomasLecocq** [comment]
The jupyter error could have been caught if we were running jupyter notebook tests, e.g.

https://gist.github.com/lheagy/f216db7220713329eb3fc1c2cd3c7826#file-test_notebooks-py

File: obspy/io/gse2/src/GSE_UTI/gse_functions.c [hunk 1]
```diff
@@ -207,7 +207,7 @@ int decomp_6b_buffer (int n_of_samples, int32_t *dta, char * (* reader)(char *,
   int i, ibuf=-1, k, inn, jsign=0, joflow=0;
   int32_t itemp;
   char cbuf[83]=" ";
-
+  setbuf(stdout, NULL);
```
Comment (barsch) [inline obspy/io/gse2/src/GSE_UTI/gse_functions.c]: unfortunatly that didn't do much - I still had multiple env's showing the message

**Comment by barsch** [comment]
https://github.com/obspy/obspy/pull/1885/commits/510baa4eaddba0418f2bf8ae058a219ed9bf526c seems to solve the output catching of GSE2 for me on all win archs (py2.7,3.4,3.5,3.6 each 32bit/64bit) - console as well as in eclipse

**Comment by barsch** [comment]
concerning Jupyter test: don't know if this is really needed as it only applies for an edge case: windows/py2.6/running Jupyter ...if you want to add it, go ahead - but make it completely optional - I don't want Jupyter as a test dependency ;)

**Comment by ThomasLecocq** [comment]
@barsch I'm always scared to push the button, so feel free to merge the PR :) you did most of the work, you deserve that "satisfaction" :)
